ReSwift/CoreTypes/Store.swift

```diff
@@ -155,3 +186,10 @@ open class Store<State: StateType>: StoreType {
         _ actionCreatorCallback: @escaping ((ActionCreator) -> Void)
     ) -> Void
 }
+
+extension Store where State: Equatable {
+    open func subscribe<S: StoreSubscriber>(_ subscriber: S)
+        where State: Equatable, S.StoreSubscriberStateType == State {
```

Comment:
Compiler segfaults if its moved into the main store body.

But yes you're right the second `where State: Equatable` is redundant. I'll remove that.